src/style/foundation/video-calling.less

```diff
@@ -141,8 +141,8 @@
     left: 50%;
     display: grid;
     padding: 0.4rem;
-    border-radius: 1rem;
-    background-color: var(--app-bg-secondary);
+    border-radius: 12px;
+    background-color: var(--inactive-call-button-bg);
```

Comment:
referred to as "background/surface variant" in figma

Would be better to rename the variable but I'm tempted to wait for a system to be drafted (a lone `calling-surface-secondary` would be a bit weird by itself)